test/caption-stream.test.js

```diff
@@ -514,6 +701,197 @@ QUnit.skip('applies preamble address codes', function() {
   QUnit.ok(false, 'not implemented');
 });
 
+QUnit.skip('applies mid-row colors', function() {
+  QUnit.ok(false, 'not implemented');
+});
+
+QUnit.test('applies mid-row underline', function() {
+  var captions = [];
+  cea608Stream.on('data', function(caption) {
+    captions.push(caption);
+  });
+
+  var packets = [
+    { ccData: 0x1425, type: 0 },
+    { ccData: characters('no'), type: 0 },
+    { ccData: 0x1121, type: 0 },
+    { ccData: characters('ye'), type: 0 },
+    { ccData: characters('s.'), type: 0 },
+    { ccData: 0x142d, type: 0 }
+  ];
+
+  packets.forEach(cea608Stream.push, cea608Stream);
+  QUnit.equal(captions[0].text, 'no <u>yes.</u>', 'properly closed by CR');
+});
+
+QUnit.test('applies mid-row italics', function() {
+  var captions = [];
+  cea608Stream.on('data', function(caption) {
+    captions.push(caption);
+  });
+
+  var packets = [
+    { ccData: 0x1425, type: 0 },
+    { ccData: characters('no'), type: 0 },
+    { ccData: 0x112e, type: 0 },
+    { ccData: characters('ye'), type: 0 },
+    { ccData: characters('s.'), type: 0 },
+    { ccData: 0x142d, type: 0 }
+  ];
+
+  packets.forEach(cea608Stream.push, cea608Stream);
+  QUnit.equal(captions[0].text, 'no <i>yes.</i>', 'properly closed by CR');
+});
+
+QUnit.test('applies mid-row italics underline', function() {
+  var captions = [];
+  cea608Stream.on('data', function(caption) {
+    captions.push(caption);
+  });
+
+  var packets = [
+    // RCL, resume caption loading
+    { ccData: 0x1425, type: 0 },
+    { ccData: characters('no'), type: 0 },
+    { ccData: 0x112f, type: 0 },
+    { ccData: characters('ye'), type: 0 },
+    { ccData: characters('s.'), type: 0 },
+    { ccData: 0x142d, type: 0 }
+  ];
+
+  packets.forEach(cea608Stream.push, cea608Stream);
+  QUnit.equal(captions[0].text, 'no <i><u>yes.</u></i>', 'properly closed by CR');
+});
+
+QUnit.test('applies PAC color underline', function() {
```

Comment:
Bit confused here. Is the color portion of this (and the following 2 tests) being tested asserted here? Your PR description mentions that color is not implemented yet. Are these tests because only white text is supported?